packages/react-form-state/src/components/Nested.tsx

```diff
@@ -23,7 +23,10 @@ export default class Nested<Fields> extends React.PureComponent<
     const innerFields: FieldDescriptors<Fields> = mapObject(
       value,
       (value, fieldPath) => {
-        const initialFieldValue = initialValue[fieldPath];
+        let initialFieldValue: any;
+        if (initialValue) {
+          initialFieldValue = initialValue[fieldPath];
+        }
```

Comment:
Would probably be a bit cleaner to do `const initialFieldValue = initialValue && initialValue[fieldPath]`

Reply:
I am a bit confused how this could ever happen if the field type is not an optional type; that is, in the example case you provided, why is `initialValue` undefined? It looks like `initialValue` should be `{brand: 'brandName1'}`, no?

Reply:
I am not sure - but I think it's du to the fact that an added list item is not present in the initial data passed to the FormState.

Reply:
In the example you provided (very helpful btw), the initial values are definitely present. I wonder if this is really more of a bug in `List` not passing down the value to the render prop correctly? Can you log out what `fields.description` is?

Reply:
The error happens only when you add a new element to the list - this new, second item is not present in the data initially passed to `<FormState initialValues=..`, which has only one element in the variants` list